ci-operator/step-registry/aro/ocm/register/aro-ocm-register-ref.yaml

```diff
@@ -0,0 +1,13 @@
+ref:
+  as: aro-ocm-register
```

Comment:
I don't see any workflows using this step.

Reply:
That is correct for now, this is a step we have to add to get ARO clusters registered into OCM. I'll see what I can find.